src/frontend/Komponenter/Behandling/Aktivitet/AktivitetArbeid/AktivitetArbeid.tsx

```diff
@@ -45,7 +45,7 @@ export const AktivitetArbeid: React.FC<VilkårProps> = ({
                     >
                         <VilkårpanelInnhold>
                             {{
-                                venstre: grunnlag.aktivitet && (
+                                venstre: grunnlag.aktivitet && skalViseSøknadsdata && (
```

Comment:
Kan kanskje fjerne `skalViseSøknadsdata` fra `AktivitetArbeidInfo` hvis den alltid er false :-)

Reply:
Tenkte å lage en oppgave på å se over disse. Det sjekkes på veldig mange nivåer innover i komponenten også i stedet for å bare sjekkes på ytterste nivå. Så måtte legge på den her for å stoppe komponenten fra å returnere en tom div. 

Tror den bare er false på saker som ikke er førstegangsbehandling

Reply:
Når behandlingen baserer seg på en søknad så er den der `true`, vel?

Reply:
Høres fint ut 👍 
Ja, menade true ja :)